### ISSUE TITLE
PHP Error

### ISSUE THREAD

**Issue by AndrewMac** [verbb/formie#868]
**Description**
I have started to get the following error, which crashes both the site and the control panel.
This is on my local dev environment at the moment.

Thanks in advance for any help or advice you may be able to give with this.

```
Exception 'yii\base\UnknownPropertyException' with message 'Setting unknown property: verbb\formie\fields\formfields\Checkboxes::matchField'

in /var/www/html/vendor/yiisoft/yii2/base/Component.php:209

Stack trace:
#0 /var/www/html/vendor/yiisoft/yii2/BaseYii.php(558): yii\base\Component->__set('matchField', NULL)
#1 /var/www/html/vendor/yiisoft/yii2/base/BaseObject.php(107): yii\BaseYii::configure(Object(verbb\formie\fields\formfields\Checkboxes), Array)
#2 [internal function]: yii\base\BaseObject->__construct(Array)
#3 /var/www/html/vendor/yiisoft/yii2/di/Container.php(419): ReflectionClass->newInstanceArgs(Array)
#4 /var/www/html/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build('verbb\\formie\\fi...', Array, Array)
#5 /var/www/html/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get('verbb\\formie\\fi...', Array, Array)
#6 /var/www/html/vendor/craftcms/cms/src/Craft.php(61): yii\BaseYii::createObject(Array, Array)
#7 /var/www/html/vendor/craftcms/cms/src/helpers/Component.php(111): Craft::createObject(Array)
#8 /var/www/html/vendor/craftcms/cms/src/services/Fields.php(600): craft\helpers\Component::createComponent(Array, 'craft\\base\\Fiel...')
#9 /var/www/html/vendor/craftcms/cms/src/services/Fields.php(625): craft\services\Fields->createField(Array)
#10 /var/www/html/vendor/craftcms/cms/src/services/Fields.php(654): craft\services\Fields->_fields(NULL)
#11 /var/www/html/vendor/craftcms/cms/src/elements/db/ElementQuery.php(2207): craft\services\Fields->getAllFields()
#12 /var/www/html/vendor/craftcms/cms/src/elements/db/ElementQuery.php(1486): craft\elements\db\ElementQuery->customFields()
#13 /var/www/html/vendor/yiisoft/yii2/db/QueryBuilder.php(227): craft\elements\db\ElementQuery->prepare(Object(craft\db\mysql\QueryBuilder))
#14 /var/www/html/vendor/yiisoft/yii2/db/Query.php(157): yii\db\QueryBuilder->build(Object(craft\elements\db\EntryQuery))
#15 /var/www/html/vendor/yiisoft/yii2/db/Query.php(287): yii\db\Query->createCommand(Object(craft\db\Connection))
#16 /var/www/html/vendor/craftcms/cms/src/db/Query.php(167): yii\db\Query->one(NULL)
#17 /var/www/html/vendor/craftcms/cms/src/elements/db/ElementQuery.php(1654): craft\db\Query->one(NULL)
#18 /var/www/html/vendor/craftcms/cms/src/services/Elements.php(563): craft\elements\db\ElementQuery->one()
#19 /var/www/html/vendor/craftcms/cms/src/services/Elements.php(483): craft\services\Elements->_elementById('id', 2, 'craft\\elements\\...', 1, Array)
#20 /var/www/html/vendor/craftcms/cms/src/services/Elements.php(631): craft\services\Elements->getElementById(2, 'craft\\elements\\...', 1)
#21 /var/www/html/vendor/nystudio107/craft-seomatic/src/services/MetaContainers.php(481): craft\services\Elements->getElementByUri('__home__', 1, true)
#22 /var/www/html/vendor/nystudio107/craft-seomatic/src/services/MetaContainers.php(379): nystudio107\seomatic\services\MetaContainers->setMatchedElement('', 1)
#23 /var/www/html/vendor/nystudio107/craft-seomatic/src/twigextensions/SeomaticTwigExtension.php(54): nystudio107\seomatic\services\MetaContainers->loadMetaContainers('/', NULL)
#24 /var/www/html/vendor/twig/twig/src/ExtensionSet.php(329): nystudio107\seomatic\twigextensions\SeomaticTwigExtension->getGlobals()
#25 /var/www/html/vendor/twig/twig/src/Environment.php(933): Twig\ExtensionSet->getGlobals()
#26 /var/www/html/vendor/nystudio107/craft-autocomplete/src/generators/AutocompleteVariableGenerator.php(76): Twig\Environment->getGlobals()
#27 /var/www/html/vendor/nystudio107/craft-autocomplete/src/generators/AutocompleteVariableGenerator.php(54): nystudio107\autocomplete\generators\AutocompleteVariableGenerator::generateInternal()
#28 /var/www/html/vendor/nystudio107/craft-autocomplete/src/Autocomplete.php(128): nystudio107\autocomplete\generators\AutocompleteVariableGenerator::generate()
#29 [internal function]: nystudio107\autocomplete\Autocomplete->generateAutocompleteClasses(Object(yii\base\Event))
#30 /var/www/html/vendor/yiisoft/yii2/base/Event.php(312): call_user_func(Array, Object(yii\base\Event))
#31 /var/www/html/vendor/yiisoft/yii2/base/Component.php(642): yii\base\Event::trigger('craft\\services\\...', 'afterLoadPlugin...', Object(yii\base\Event))
#32 /var/www/html/vendor/craftcms/cms/src/services/Plugins.php(278): yii\base\Component->trigger('afterLoadPlugin...')
#33 /var/www/html/vendor/craftcms/cms/src/base/ApplicationTrait.php(1487): craft\services\Plugins->loadPlugins()
#34 /var/www/html/vendor/craftcms/cms/src/console/Application.php(48): craft\console\Application->_postInit()
#35 /var/www/html/vendor/yiisoft/yii2/base/BaseObject.php(109): craft\console\Application->init()
#36 /var/www/html/vendor/yiisoft/yii2/base/Application.php(204): yii\base\BaseObject->__construct(Array)
#37 /var/www/html/vendor/yiisoft/yii2/console/Application.php(89): yii\base\Application->__construct(Array)
#38 [internal function]: yii\console\Application->__construct(Array)
#39 /var/www/html/vendor/yiisoft/yii2/di/Container.php(419): ReflectionClass->newInstanceArgs(Array)
#40 /var/www/html/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build('craft\\console\\A...', Array, Array)
#41 /var/www/html/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get('craft\\console\\A...', Array, Array)
#42 /var/www/html/vendor/craftcms/cms/src/Craft.php(61): yii\BaseYii::createObject(Array, Array)
#43 /var/www/html/vendor/craftcms/cms/bootstrap/bootstrap.php(247): Craft::createObject(Array)
#44 /var/www/html/vendor/craftcms/cms/bootstrap/console.php(51): require('/var/www/html/v...')
#45 /var/www/html/craft(26): require('/var/www/html/v...')
#46 {main}
```

**Steps to reproduce**
1. 
2.

### Form settings
Please pick from the options below.

- Multi-page form: Yes or No
- Submission Method: Ajax or Page Reload
- Client-side Validation: Yes or No
- Custom Form Templates: Yes or No

**Additional info**
- Plugin version: 1.4.28
- Craft version: 3.7.40.1 Pro
- Multi-site: Yes
**Additional context**

**Comment by engram-design** [comment]
Was this version maybe back-tracked? The `matchField` setting was only introduced in `1.5.0`.

**Comment by AndrewMac** [comment]
Brilliant, yes! I did have a problem on my local dev site recently, and had to do quite a lot of fixing to get things working again. And it does look like Formie got downgraded during that process. I have now managed to get it back up to date, although I had to do resolve a few composer incompatibilities.

Thanks so much, once again, and as ever. I hope this didn't take up more than a few seconds of your valuable time!

Cheers, Andrew
